zero/gkyl_gk_geometry_mirror.h

```diff
@@ -3,9 +3,8 @@
 #include <gkyl_gk_geometry.h>
 /**
  * Create a new geometry object using mirror input (efit)
+ * Wrapper for gkyl_gk_geometry_mirror_advance, neccisary for the non-uniform grids
```

Comment:
This added comment seems misleading. Isn't this used/needed if we are not using nonuniform grids? To call it a wrapper for nonuniform grids seems wrong.